src/python/pants/backend/jvm/tasks/jvm_compile/execution_graph.py

```diff
@@ -28,13 +28,15 @@ def __init__(self, key, fn, dependencies, size=0, on_success=None, on_failure=No
     :param on_success: Zero parameter callback to run if job completes successfully. Run on main
                        thread.
     :param on_failure: Zero parameter callback to run if job completes successfully. Run on main
-                       thread."""
+                       thread.
+    :param run_asap: Boolean indicating whether or not to queue job immediately once unblocked."""
```

Comment:
If this ends up needing another CI run, it would be good to add an explanation of how this relates to the `size`.